src/test/java/com/exasol/adapter/sql/SqlNodeTest.java

```diff
@@ -21,7 +20,11 @@ void testToSimpleSql() {
         final String expectedSql = "SELECT \"USER_ID\", COUNT(\"URL\") FROM \"CLICKS\"" + " WHERE 1 < \"USER_ID\"" //
                 + " GROUP BY \"USER_ID\"" + " HAVING 1 < COUNT(\"URL\")" + " ORDER BY \"USER_ID\"" + " LIMIT 10";
         final String actualSql = node.toSimpleSql();
-        assertEquals(SqlTestUtil.normalizeSql(expectedSql), SqlTestUtil.normalizeSql(actualSql));
+        assertEquals(normalizeSql(expectedSql), normalizeSql(actualSql));
```

Comment:
I think that the normalization step is not necessary here if you use `JSONAssert` instead.

Reply:
fixed